build-and-test.ps1

```diff
@@ -17,6 +17,7 @@ else {
 
 $manifest = Get-Content "manifest.json" | ConvertFrom-Json
 $manifestRepo = $manifest.Repos[0]
+$(docker version) | % { Write-Host "$_" }
```

Comment:
IMO this should not be in the middle of the variable block.  I think this would better belong around line 10 or 17 with a blank link before and after.

Reply:
Alright. Moved around line 10.